### PR TITLE
Feat/card

### PR DESCRIPTION
**Pull request by KatharinaWuerth**
Please checkout my branch feat/card

### CODE REVIEW

File: setup-server.js [hunk 1]
```diff
@@ -1,10 +1,27 @@
+/**
+ * You should not need to change this
+ */
+
 const express = require('express');
 const path = require('path');
 const mongoose = require('mongoose');
+const dotenv = require('dotenv');
+const fs = require('fs');
+dotenv.config();
```
Comment (vmarkwardt) [inline setup-server.js]: clever, where did you get that from?

File: src/app/App.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import styled from 'styled-components'
```
Comment (vmarkwardt) [inline src/app/App.js]: I cannot see the whole file, do you need styled-components in App.js

File: src/app/AppGrid.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components'
+
+export default styled.div`
+  display: grid;
+  grid-template-columns: 60px auto 60px;
+  grid-template-rows: auto;
+  position: absolute;
```
Comment (vmarkwardt) [inline src/app/AppGrid.js]: Do you need explicit positioning? I am not sure.

File: src/cards/Card.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  box-shadow: 2px 2px 9px -1px darkgray;
+  margin: 20px 0;
+`
+
+const Image = styled.img`
+  width: 100%;
+`
+
+const StyledName = styled.h3`
+  margin-block-end: 0;
```
Comment (vmarkwardt) [inline src/cards/Card.js]: I am not sure, what does that do?

File: src/cards/Card.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+  width: 30%;
+  display: inline-flex;
+  text-align: left;
+  margin: 0;
+`
+
+export default function Card({ img, title }) {
+  return (
+    <StyledCard>
+      <Image src={testPicture} img={img} />
```
Comment (vmarkwardt) [inline src/cards/Card.js]: I would suggest an alt-attribute,
what does the img-attribute do?

File: src/cards/Card.js [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+`
+
+const StyledName = styled.h3`
+  margin-block-end: 0;
+`
+
+const StyledTextbox = styled.section`
+  padding: 20px;
+`
+const StyledLine = styled.hr`
```
Comment (vmarkwardt) [inline src/cards/Card.js]: This looks nice, do you still need a line.png file? consider deleting it.

File: src/index.js [hunk 7]
```diff
@@ -1,11 +1,11 @@
-import React from 'react';
```
Comment (vmarkwardt) [inline src/index.js]: These changes result probably out of creating the `.prettierrc` -file

File: setup-server.js (hunk 1 above)
Comment (KatharinaWuerth) [reply setup-server.js]: This is from the setup that we got yesterday.

File: src/app/App.js (hunk 2 above)
Comment (KatharinaWuerth) [reply src/app/App.js]: styled-components in App.js are still a work in progress

**Pull request merged**